EIPS/eip-4906.md

```diff
@@ -33,26 +35,32 @@ The **metadata update extension** is OPTIONAL for ERC-721 contracts.
 interface IERC4906 is IERC165{
     /// @dev This event emits when the metadata of a token is changed.
     /// So that the third-party platforms such as NFT market could
-    /// timely update the images and related attributes of the NFT
-    event MetadataUpdate(uint256 _tokenId);
+    /// timely update the images and related attributes of the NFT.
+    /// Alternately, this event may be emitted to request that third-party
+    /// platforms automatically refresh the metadata on a specified schedule.
+    event MetadataUpdate(uint256 _fromTokenId, uint256 _toTokenId, uint256 _refreshIntervalSeconds);
```

Comment:
suggestion : 
change `event MetadataUpdate(uint256 _fromTokenId, uint256 _toTokenId, uint256 _refreshIntervalSeconds);` to 
 `event BatchMetadataUpdate(uint256 _fromTokenId, uint256 _toTokenId);`

Reply:
Sounds good!  Do you want me to leave two events?  The original MetadataUpdate(uint256 _tokenId) and the additional BatchMetadataUpdate(uint256 _fromTokenId, uint256 _toTokenId)?

Also - curious to know your thoughts on a polling mechanism - I have seen several Dynamic NFTs that could benefit from re-caching metadata at a specified interval.

Reply:
> Do you want me to leave two events? The original MetadataUpdate(uint256 _tokenId) and the additional BatchMetadataUpdate(uint256 _fromTokenId, uint256 _toTokenId)?

Yes.

Reply:
> > เขียงและได้สองอย่างMetadataUpdate ดั้งเดิม (uint256 _tokenId) BatchMetadataUpdate เพิ่มเติม (uint256 _fromTokenId, uint256 _toTokenId)
> 
> ใช่.

เปลี่ยนตรงไหรค้ะ